### PR TITLE
Feature/login screen

### PR DESCRIPTION
**Pull request by kalibria**
Create login form using MaterialUI component and Formik to control inputs. After entering login and username a request is made to server. If login or username is incorrect user see warning. If username and password are correct sessionID from the response is recorded in Local Storage.

### CODE REVIEW

File: src/services/hondaApi.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const hondaApi = createApi({
+  reducerPath: 'hondaApi',
+  baseQuery: fetchBaseQuery({
+    baseUrl: 'https://18qv4o7xm9.execute-api.eu-west-1.amazonaws.com',
```
Comment (ikushlianski) [inline src/services/hondaApi.tsx]: please extract information that can vary into environment variables

Comment (ikushlianski) [reply src/services/hondaApi.tsx]: these env variables should live in `.env` files and should not be under version controls, cause they potentially contain secrets like logins, passwords etc.

File: src/utils/queryProcessing.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+class queryProcessing {
+  errorStatus(error: FetchBaseQueryError | SerializedError) {
+    if (
+      'data' in error &&
+      typeof error.data === 'object' &&
+      error.data !== null &&
+      'status' in error.data &&
+      typeof error.data.status === 'string'
+    ) {
+      localStorage.removeItem('sessionID');
```
Comment (ikushlianski) [inline src/utils/queryProcessing.tsx:13]: I'd name the LS key something like 'isAuthenticated'

File: src/services/hondaApi.tsx (hunk 1 above)
Comment (kalibria) [reply src/services/hondaApi.tsx]: added environment variables

File: src/utils/queryProcessing.tsx (hunk 2 above)
Comment (kalibria) [reply src/utils/queryProcessing.tsx:13]: in the next PR LS key has already been renamed

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**